presto-main/src/main/java/com/facebook/presto/spiller/BinaryFileSingleStreamSpiller.java

```diff
@@ -27,44 +28,34 @@
 
 import javax.annotation.concurrent.NotThreadSafe;
 
-import java.io.BufferedInputStream;
-import java.io.BufferedOutputStream;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.util.Iterator;
-import java.util.List;
-import java.util.stream.IntStream;
-import java.util.stream.Stream;
 
 import static com.facebook.presto.execution.buffer.PagesSerdeUtil.writePage;
 import static com.facebook.presto.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
-import static com.facebook.presto.util.ImmutableCollectors.toImmutableList;
 import static com.google.common.base.Preconditions.checkState;
-import static com.google.common.util.concurrent.Futures.immediateFuture;
 import static java.util.Objects.requireNonNull;
-import static java.util.stream.Collectors.toList;
 
 @NotThreadSafe
-public class BinaryFileSpiller
-        implements Spiller
+public class BinaryFileSingleStreamSpiller
```

Comment:
I don't know. I followed naming pattern already in place. Shorter name seems fine. Renaming.